app/domain/authentication/authn_jwt/input_validation/parse_mandatory_claims.rb

```diff
@@ -0,0 +1,83 @@
+module Authentication
+  module AuthnJwt
+    module InputValidation
+      # Parse mandatory-claims secret value and return a validated claims list
+      ParseMandatoryClaims ||= CommandClass.new(
+        dependencies: {
+          validate_claim_name: ValidateClaimName.new(
```

Comment:
I think only the class should be in dependency and the call to .new should be in runtime.

Add MANDATORY_CLAIMS_DENY_LIST to a dependency also

Reply:
will always call to the new (when it possible)  in the dependencies section

Reply:
But what if we dynammicly want to put the un premmited list?

Reply:
From the policy for example

Reply:
So you will create this instance with different  value